src/internal/HtmlColorToString.cpp

```diff
@@ -0,0 +1,52 @@
+/*-------------------------------------------------------------------------
```

Comment:
did you test it? I merged it all into one cpp, it was smart enough to only bring the code that was used.